frontend/src/components/EditAdminTeam.js

```diff
@@ -72,83 +72,88 @@ export default function EditAdminTeam({ user }) {
 		setAreYouSureChecked(false);
 	};
 
-	return <Grid
-		container
-		spacing={2}
-		alignItems="center"
-		justifyContent="center"
-		direction="column">
-		<Grid item style={{ width: "100%" }}>
-			<Grid container justify="space-between" style={{ width: "100%" }}>
-				<ParagraphTypography variant="body1" align="left" gutterBottom>
-					Search for a user by their email and remove them from the admins team or send an invite to someone to join the admin teams.
-				</ParagraphTypography>
-			</Grid>
-			<Box component="form" onSubmit={checkIfUserIsInAdminTeam} noValidate sx={{ mt: 1 }} >
-				<TextField
-					margin="normal"
```

Comment:
Should we reduce the number of lines here?
Like:
```js
<TextField
  margin="normal" fullWidth  required autoFocus
  id="email" label="Email" name="email"
/>
```

Reply:
I would even go for a single line, I find this best
```js
<TextField margin="normal" fullWidth required autoFocus id="email" label="Email" name="email"/>
```
This is not a long line (comparing it to C++ for example where developers often use quite wide screens).
But this is your preference.

Reply:
I will change some spread apart components in my next PR.